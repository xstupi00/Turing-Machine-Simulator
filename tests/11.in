S a A b
A b C d
A b D f
C d E g
C d D e
D e F x
a