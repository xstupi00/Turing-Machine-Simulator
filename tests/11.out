Sa
Ab
Cd
De
Fx
